src/cmd/root.go

```diff
@@ -1,5 +1,8 @@
+//go:build linux && cgo && libsubid
+// +build linux,cgo,libsubid
```

Comment:
We might not need these tags.  We always just assume Linux and CGO.  We could also make `libsubuid` a mandatory dependency.